nautobot/ui/src/views/generic/GenericView.js

```diff
@@ -93,7 +93,6 @@ export default function GenericView({
                 if (pathname === "/" || !isSuccess || !menu) {
                     return [
                         {
-                            children: "Home",
                             key: `0_home`,
                             type: "text",
```

Comment:
What's the purpose/benefit of leaving the `key` and `type` here versus just returning an empty object, empty list or null?